config/vendor_api/draft.yml

```diff
@@ -103,3 +109,84 @@ components:
           description: Optional details about why the application was rejected for the given reason.
           example: You did not attend any of the arranged interviews.
           maxLength: 65535
+    HESAData:
+      type: object
+      description: |
+        Values to populate this candidate’s HESA Initial Teacher
+        Training data return.  Available once an offer has been accepted.
+      required:
+        - sex
+        - disability
+        - ethnicity
+      properties:
+        sex:
+          type: string
+          nullable: true
+          description: The candidate’s sex as a [2-digit HESA code for Sex](https://www.hesa.ac.uk/collection/c22053/e/sexid)
+          example: "10"
+          enum:
+          - "10"
+          - "11"
+          - "12"
+        disability:
+          nullable: true
+          type: array
+          items:
+            type: string
+          description: The candidate’s disabilities as an array of [2-digit HESA codes for Disability](https://www.hesa.ac.uk/collection/c22053/e/disable)
+          example:
+            - "95"
+          enum:
+          - "95"
+          - "58"
+          - "57"
+          - "59"
+          - "51"
+          - "54"
+          - "55"
+          - "56"
+          - "53"
+          - "96"
+          - "98"
+          - "99"
+        ethnicity:
+          type: string
+          nullable: true
+          description: The candidate’s ethnicity as [a 3-digit HESA code for Ethnicity](https://www.hesa.ac.uk/collection/c22053/e/ethnic)
+          example: "180"
+          enum:
+            - "180"
+            - "100"
+            - "101"
+            - "103"
+            - "104"
+            - "119"
+            - "120"
+            - "121"
+            - "139"
+            - "140"
+            - "141"
+            - "142"
+            - "159"
+            - "160"
+            - "163"
+            - "166"
+            - "168"
+            - "179"
+            - "899"
+            - "997"
+            - "998"
+            - "999"
```

Comment:
As above, should we use `998` for Prefer not to say?

What would we return for a candidate who applied last year?